FinSearchUnified/Tests/Helper/StaticHelperTest.php

```diff
@@ -55,6 +54,125 @@ protected function tearDown()
         Shopware()->Container()->load('session');
         Shopware()->Container()->reset('config');
         Shopware()->Container()->load('config');
+        Shopware()->Container()->reset('fin_search_unified.config_loader');
+        Shopware()->Container()->load('fin_search_unified.config_loader');
+    }
+
+    /**
+     * Data provider for checking isFindologicActive behavior
+     *
+     * @return array
+     */
+    public function isFindologicActiveDataprovider()
+    {
+        return [
+            'Plugin is deactivated' => [
+                'ActivateFindologic' => false,
+                'ShopKey' => '0000000000000000ZZZZZZZZZZZZZZZZ',
+                'QueryParameter' => null,
+                'Staging' => false,
+                'ConfigStaging' => true,
+
+            ],
+            'FINDOLOGIC is active and shopkey is empty' => [
+                'ActivateFindologic' => true,
+                'ShopKey' => '',
+                'QueryParameter' => null,
+                'Staging' => false,
+                'ConfigStaging' => true,
+
+            ],
+            'FINDOLOGIC is active without query parameter' => [
+                'ActivateFindologic' => true,
+                'ShopKey' => '0000000000000000ZZZZZZZZZZZZZZZZ',
+                'QueryParameter' => null,
+                'Staging' => false,
+                'ConfigStaging' => true,
+
+            ],
+            'FINDOLOGIC is active with query parameter' => [
+                'ActivateFindologic' => true,
+                'ShopKey' => '0000000000000000ZZZZZZZZZZZZZZZZ',
+                'QueryParameter' => 'on',
+                'Staging' => true,
+                'ConfigStaging' => false,
+
```

Comment:
I am missing some cases such as:
* Plugin inactive, shopkey set, not staging mode, no query param.  

Or successful cases even:
* Plugin active, shopkey set, not staging mode, no query param.